receipt_label/receipt_label/utils/chroma_hash.py

```
@@ -0,0 +1,315 @@
+                    total_size += file_size
+        
+        # Sort files by relative path for deterministic ordering
+        all_files.sort(key=lambda x: x[0])
+        
+        logger.info("Found %d files, total size: %d bytes", len(all_files), total_size)
+        
+        # Calculate individual file hashes and combine them
+        hash_obj = hashlib.new(algorithm)
+        processed_files = []
+        
+        for relative_path, full_path, file_size in all_files:
```

Ruff (0.12.2)

Loop control variable `file_size` not used within loop body

(B007)